### ISSUE TITLE
travis failed on master / not running

### ISSUE THREAD

**Issue by belm0** [HyperionGray/trio-websocket#85]
* CI doesn't appear to be running for my last two PR's (#83, #84 )
  * travis build looks failed on master
  * build badge in README is not pointing to master

**Comment by belm0** [comment]
https://travis-ci.org/HyperionGray/trio-websocket/jobs/453122094

```
$ make test
pytest --cov=trio_websocket
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/pluggy/__init__.py", line 397, in load_setuptools_entrypoints
    plugin = ep.load()
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2404, in load
    self.require(*args, **kwargs)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 2427, in require
    items = working_set.resolve(reqs, env, installer, extras=self.extras)
  File "/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages/pkg_resources/__init__.py", line 875, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.VersionConflict: (pytest 3.3.0 (/home/travis/virtualenv/python3.6.3/lib/python3.6/site-packages), Requirement.parse('pytest>=3.6'))
```

I have no idea why travis isn't running on those two PR's.

**Comment by mehaase** [comment]
I think it didn't build because I changed Travis settings. I had both the "branch" and "PR" triggers turned on, but it was building twice for each PR, so I turned off the PR trigger. But I guess the branch trigger doesn't apply to external repos...

<img width="825" alt="screen shot 2018-11-10 at 1 03 43 pm" src="https://user-images.githubusercontent.com/320904/48304561-4622d180-e4e9-11e8-8bf2-dd45f53162b2.png">

I've turned the PR trigger back on just now.
